source/data/local_migration.rst

```diff
@@ -0,0 +1,261 @@
+and calculate usage.  One example command is:
+
+.. code-block:: shell
+
+        tree --du -h -d -L 2 --sort=size DIRECTORY
+
+- ``--du`` will calculate disk usage on directories
+- ``-h`` will display human-readable (K,M,G,T) volumes
+- ``-d`` will summarize directories
+- ``-L 2`` will only show two levels of directories
+- ``--sort=size`` will sort output by size
+
+.. code-block:: shell
+
+    % tree --du -h -d --sort=size -L 2 .
+    [8.8K]  .
+    ├── [6.3K]  source
+    │   ├── [2.6K]  images
+    │   ├── [ 416]  data
+    │   ├── [ 416]  systems
+    │   ├── [ 288]  software
+    │   ├── [ 224]  slurm
+    │   ├── [ 192]  _templates
+    │   ├── [ 192]  accounts
+    │   ├── [ 160]  _downloads
+    │   ├── [ 160]  files
+    │   ├── [ 128]  _search
+    │   ├── [ 128]  _static
+    │   ├── [ 128]  contributing
+    │   ├── [ 128]  help
+    │   ├── [ 128]  logging_in
+    │   ├── [  96]  FAQ
+    │   ├── [  96]  compilers
+    │   ├── [  96]  connecting
+    │   └── [  96]  queue_policy
+    ├── [1.7K]  build
+    │   ├── [ 992]  html
+    │   └── [ 608]  doctrees
+    └── [  96]  utils
+
+      15K used in 24 directories
+
+rsync
+-----
+
+For basic migration, it is recommended to use the ``rsync`` tool to
+transfer the files and directories. One example command is:
+
+.. code-block:: shell
+
+    rsync --archive --verbose --one-file-system /full/path/to/source/directory/ /full/path/to/destination/directory
+
+.. warning::
+
+    It is very important that you have a trailing slash after the
+    source directory: ``/full/path/to/source/directory/`` **/**. If you do not,
+    a second invocation of the same command will attempt to retransfer all of
+    the data into a subdirectory, for example:
+
+    ``/full/path/to/source/directory/directory``.
+
+- ``--archive`` (``-a``) will ensure all ownership and dates are
+  preserved in the transfer.
+- ``--verbose`` (``-v``) will display details of every file being
+  transfered. If you have lots of small files, this will slow down the
+  transfer processes.
+- ``--one-file-system`` (``-x``) restricts the transfer to the source
+  filesystem. This is important when symlinks are used to point to
+  data that exists on other filesystems.
+
+To keep the two directories exactly the same, use ``--delete`` -- if
+the file **did not** exist in source, you want it removed on
+destination if does exist:
+
+- ``--delete`` means to remove files from the destination that are not
+  in the source directory. If after a completed rsync a file was then
+  removed from the source, then the next rsync with the –delete option
+  would then remove the file from the destination/ It may be
+  preferable to clean up the source only after confirming that all the
+  files have been transferred.
+
+.. warning::
+
+    Do not use the ``–-delete`` option if you do not want data in the destination
+    directory to be removed.
+
+xsync
+-----
+
+On Jet and perhaps Hera, an additional data synchronization tool,
```

Comment:
**_"Perhaps"_** Hera?  

Can we confirm that?  Or do you mean it is only on certain nodes?